src/test/java/org/folio/rest/impl/ApiTestBase.java

```diff
@@ -142,16 +143,29 @@ Response verifyGet(String url, String expectedContentType, int expectedCode) {
   }
 
   Response verifyGet(String url, Headers headers, String expectedContentType, int expectedCode) {
-    return RestAssured
-      .with()
-        .headers(headers)
+    return verifyGet(RestAssured.with()
+      .headers(headers), url, expectedContentType, expectedCode);
+  }
+
+  Response verifyGetWithParam(String url, String expectedContentType, int expectedCode, String paramName, String paramValue) {
+    Headers headers = prepareHeaders(X_OKAPI_URL, X_OKAPI_TENANT);
+    return verifyGet(RestAssured.with()
+      .headers(headers)
+      .queryParam(paramName, paramValue),
```

Comment:
Could you please move this `RequestSpecification` to a local variable for better readability?